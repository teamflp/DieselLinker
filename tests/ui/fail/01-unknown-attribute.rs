use diesel_linker::relation;

#[relation(model = "Post", wrong_attr = "some_value")]
struct User;

fn main() {}
