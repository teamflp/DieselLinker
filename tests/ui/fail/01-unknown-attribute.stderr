error: Unknown attribute, expected one of: `relation_type`, `model`, `fk`, `parent_primary_key`, `join_table`, `fk_parent`, `fk_child`, `method_name`, `backend`, `primary_key`, `child_primary_key`, `eager_loading`, `async`, `error_type`
 --> tests/ui/fail/01-unknown-attribute.rs:3:28
  |
3 | #[relation(model = "Post", wrong_attr = "some_value")]
  |                            ^^^^^^^^^^
